integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonShowLoadsCommand.scala

```diff
@@ -52,7 +54,9 @@ case class CarbonShowLoadsCommand(
         AttributeReference("Merged To", StringType, nullable = false)(),
         AttributeReference("File Format", StringType, nullable = false)(),
         AttributeReference("Data Size", StringType, nullable = false)(),
-        AttributeReference("Index Size", StringType, nullable = false)())
+        AttributeReference("Index Size", StringType, nullable = false)(),
+        AttributeReference("Is Sorted", StringType, nullable = false)(),
+        AttributeReference("Sort Columns", StringType, nullable = true)())
```

Comment:
2 problems here:
1. if the `Sort Columns` is not `NA` here, will `Is Sorted` always be `true`? If so, I think we can only keep the `Sorted Columns` here.
2. Since different segments can have different `sort_scope`, why do we discard the `sort scope` information here?

Reply:
2. in the segment, we lost the sort_scope information.
1. is_sorted has 3 options( true, false, NA),  but we always can get sort_columns from the schema in the index.  Sort_columns will show columns only when the is_sorted is true.